### PR TITLE
QIIME - Add multiple_split_libraries_fastq

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

I added `multiple_split_libraries_fastq` wrapper and tests.

Bérénice

### CODE REVIEW

File: tools/qiime/qiime_core/multiple_split_libraries_fastq.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <version_command>multiple_split_libraries_fastq.py -v</version_command>
+    <command detect_errors="aggressive"><![CDATA[
+        mkdir input
+        &&
+        #for $i, $s in enumerate( $input_files )
+            cp '$s.reads' input/reads_${i}.fastq
```
Comment (shiltemann) [inline tools/qiime/qiime_core/multiple_split_libraries_fastq.xml]: do we need to copy here or could we make symlinks instead?

File: tools/qiime/qiime_core/multiple_split_libraries_fastq.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            #end if
+            --read_indicator 'reads_'
+            --barcode_indicator 'barcodes_'
+            --mapping_indicator 'mapping_'
+            --mapping_extensions 'txt'
+            --leading_text '$leading_text'
+            --trailing_text '$trailing_text'
+            --sampleid_indicator '.'
+        &&
+        ls output
```
Comment (shiltemann) [inline tools/qiime/qiime_core/multiple_split_libraries_fastq.xml]: did you want to expose ls output to user or is this statement left over from debugging?

File: tools/qiime/qiime_core/multiple_split_libraries_fastq.xml (hunk 1 above)
Comment (bebatut) [inline tools/qiime/qiime_core/multiple_split_libraries_fastq.xml]: Good point. I changed it

File: tools/qiime/qiime_core/multiple_split_libraries_fastq.xml (hunk 2 above)
Comment (bebatut) [inline tools/qiime/qiime_core/multiple_split_libraries_fastq.xml]: Left over from debugging :) Removed

**Pull request merged**